docs/research.md

```diff
@@ -8,16 +8,25 @@ The Polkadot [research pages](https://research.polkadot.network) are edited dire
 researchers at W3F and contains papers, articles and documents that are sources for the information
 on the wiki.
 
-The following papers and articles may be of special interest for those interested in investigsting
+The following papers and articles may be of special interest for those interested in investigating
 Polkadot from a research or academic perspective.
 
+## Research Papers
+
 - [Overview of Polkadot and its Design Considerations](https://arxiv.org/pdf/2005.13456.pdf) - A
-  broad overview of the design of Polkadot.
+  broad overview of the design of Polkadot. Authors: Jeff Burdges, Alfonso Cevallos, Peter Czaban, Rob Habermeier, Syed Hosseini, Fabio Lama, Handan Kılınç Alper, Ximin Luo, Fatemeh Shirazi, Alistair Stewart, Gavin Wood
 - [Intro to Nominated Proof of Stake](https://research.web3.foundation/en/latest/polkadot/NPoS/index.html) -
   A description of the NPoS scheme which selects which validators are allowed to participate in the
-  consensus protocol of Polkadot.
-- [BABE](https://research.web3.foundation/en/latest/polkadot/BABE/Babe.html) - Blind Assignment for
-  Blockchain Extension, the block production mechanism of Polkadot's relay chain.
-- [GRANDPA](https://research.web3.foundation/en/latest/polkadot/GRANDPA.html) - GHOST-based
-  Recursive Ancestor Deriving Prefix Agreement, Polkadot's finality gadget. This page includes both
-  the abstract and the full GRANDPA papers.
+  consensus protocol of Polkadot. Author: Alfonso Cevallos
+- [GRANDPA: A Byzantine Finality Gadget](https://arxiv.org/abs/2007.01560) - GHOST-based  Recursive Ancestor Deriving Prefix Agreement, Polkadot's finality gadget. Authors: Alistair Stewart and Lefteris Kokoris Kogias
+- [A Verifiably Secure and Proportional Committee Election Rule](https://arxiv.org/abs/2004.12990) - Validator Election in Nominated Proof of Stake. Authors: Alfonso Cevallos and Alistair Stewart
+[Network Time with a Consensus on Clock](https://arxiv.org/abs/2007.01560) - Consensus on Clock in Universally Composable Timing. Author: Handan Kılınç Alper
+[Delay Encryption](https://eprint.iacr.org/2020/638) - Delay Encryption roughly be described as “identity based encryption with slow derived private key issuance”. Authors: Jeff Burdges and Luca de Feo
+
+## Research Blogs
```

Comment:
You need to make these list items, so start every line with a dash or asterisk. If you look at the preview of this file, you'll notice they're not rendered as a list, but inline one after the other.